test/lib/ufe/testCamera.py

```diff
@@ -0,0 +1,245 @@
+            cls.pluginsLoaded = mayaUtils.isMayaUsdPluginLoaded()
+
+    @classmethod
+    def tearDownClass(cls):
+        standalone.uninitialize()
+
+    def setUp(self):
+        ''' Called initially to set up the maya test environment '''
+        # Load plugins
+        self.assertTrue(self.pluginsLoaded)
+
+        # value from UsdGeomLinearUnits
+        self.inchesToCm = 2.54
+        self.mmToCm = 0.1
+
+    def _StartTest(self, testName):
+        cmds.file(force=True, new=True)
+        mayaUtils.loadPlugin("mayaUsdPlugin")
+        self._testName = testName
+        testFile = testUtils.getTestScene("camera", self._testName + ".usda")
+        mayaUtils.createProxyFromFile(testFile)
+        globalSelection = ufe.GlobalSelection.get()
+        globalSelection.clear()
+
+    def _TestCameraAttributes(self, ufeCamera, usdCamera):
+        # Trust that the USD API works correctly, validate that UFE gives us
+        # the same answers
+        self._TestHorizontalAperture(ufeCamera, usdCamera)
+        self._TestVerticalAperture(ufeCamera, usdCamera)
+        self._TestHorizontalApertureOffset(ufeCamera, usdCamera)
+        self._TestVerticalApertureOffset(ufeCamera, usdCamera)
+        self._TestFStop(ufeCamera, usdCamera)
+        self._TestFocalLength(ufeCamera, usdCamera)
+        self._TestFocusDistance(ufeCamera, usdCamera)
+        self._TestClippingRange(ufeCamera, usdCamera)
+        self._TestProjection(ufeCamera, usdCamera)
+
+    def _TestClippingRange(self, ufeCamera, usdCamera):
+        clippingAttr = usdCamera.GetAttribute('clippingRange')
+        clippingRange = clippingAttr.Get()
+
+        # read the nearClipPlane and check the value is what we expect.
+        self.assertAlmostEqual(ufeCamera.nearClipPlane(), clippingRange[0])
+        self.assertAlmostEqual(ufeCamera.farClipPlane(), clippingRange[1])
+
+        # setting camera values through Ufe is not yet implemented in MayaUSD
+        # ufeCamera.nearClipPlane(10)
+        # ufeCamera.farClipPlane(20)
+        # clippingRange = clippingAttr.Get()
+        # self.assertAlmostEqual(10, clippingRange[0])
+        # self.assertAlmostEqual(20, clippingRange[1])
+
+        # set the clipping planes using USD
+        clippingAttr.Set(Gf.Vec2f(1, 50))
+
+        # read the nearClipPlane and check the value is what we just set.
+        self.assertAlmostEqual(ufeCamera.nearClipPlane(), 1)
+        self.assertAlmostEqual(ufeCamera.farClipPlane(), 50)
+
+    def _TestHorizontalAperture(self, ufeCamera, usdCamera):
+        usdAttr = usdCamera.GetAttribute('horizontalAperture')
+        horizontalAperture = usdAttr.Get()
+
+        # the USD schema specifics that horizontal aperture is authored in mm
+        # the ufeCamera gives us inches
+        self.assertAlmostEqual(self.mmToCm * horizontalAperture, self.inchesToCm * ufeCamera.horizontalAperture())
+
+        # setting camera values through Ufe is not yet implemented in MayaUSD
+
+        usdAttr.Set(0.5)
```

Comment:
USD doesn't do any conversion, so the Set(0.5) isn't setting the horizontal aperture to 0.5 scene units, but to 0.5mm, the defined linear unit for horizontal aperture.